theme/horizontalrule.css

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+.ck-content .ck-horizontal-rule {
```

Comment:
cc @dkonopka Leaving this part to you.

Reply:
Ye, it was me, so self-review, great line of code, gj @dkonopka 👍 😂